config/config_test.go

```diff
@@ -102,6 +102,7 @@ var expectedConf = &Config{
 					KeyFile:  filepath.FromSlash("testdata/valid_key_file"),
 				},
 			},
+			Headers: map[string]string{"name": "value"},
```

Comment:
Would you add some test cases for the invalid headers as well to make sure all of that logic works?

Reply:
Done, thanks!